temfpy/tests/test_econometrics.py

```diff
@@ -0,0 +1,105 @@
+"""Tests for econometrics module."""
+import numpy as np
+import pandas as pd
+import temfpy.econometrics as tpe

```

Comment:
I think is preferable to just import the functions directly in this case (i.e. `from temfpy.econometrics import multinomial_probit`)

Reply:
I think this must be imported as a whole due to the getattr() function in Line 165